tokio/src/sync/semaphore_ll.rs

```diff
@@ -610,6 +610,7 @@ impl Permit {
     }
 
     /// Returns `true` if the permit has been acquired
+    #[allow(dead_code)] // may be used later
```

Comment:
I'm usually against allowing `dead_code`, but in this case, I think it is a good choice. Reason: the entire `semaphore_ll.rs` file is planed to be removed.